### PR TITLE
WIP Support fungible invites directly

### PR DESCRIPTION
**Pull request by erights**
Draft. WIP. First step not obviously related --- not yet worth looking.

### CODE REVIEW

**Comment by erights** [comment]
Sorry. **Not ready for review.** I accidentally asked for review because I thought I was on a different tab.

File: packages/ERTP/src/mathHelpers/strSetMathHelpers.js [hunk 1]
```diff
@@ -25,42 +30,98 @@ const strSetMathHelpers = harden({
...
+      } else {
+        result.push(rightStr);
+        rightI += 1;
+      }
+    }
+    if (leftI < leftLen) {
+      result.push(left[leftI]);
+    } else if (rightI < rightLen) {
+      result.push(right[rightI]);
+    }
```
Comment (erights) [inline packages/ERTP/src/mathHelpers/strSetMathHelpers.js:91]: I think these leftover processing loops should be while loops. Also the two in `doSubtract` below.

Comment (erights) [reply packages/ERTP/src/mathHelpers/strSetMathHelpers.js:91]: Hmm. not sure. need closer examination than I'm about to give it.

**Comment by katelynsills** [comment]
This was described elsewhere as "objective" fungible versions of NFTs. If so, I think we don't want that as a feature. It looks like there is a lot of work in this PR to get the amount value elements sorted, though, so that is important to keep for future work.

**Comment by erights** [comment]
Closed in favor of #2642

**Comment by erights** [comment]
Scavenged some into https://github.com/Agoric/agoric-sdk/pull/3531

**Pull request closed without merging**